config/config.go

```diff
@@ -54,7 +54,7 @@ type AppConfig struct {
 	// RequestLogging is flag for enabling logging in go-api-router.
 	RequestLogging bool `json:"request_logging" mapstructure:"request_logging"`
 	// Server is a general configuration for bux-server.
-	Server *ServerConfig `json:"server" mapstructure:"server"`
+	Server *ServerConfig `json:"server_config" mapstructure:"server_config"`
```

Comment:
Why?

Reply:
This is actually a funny case.

Because we changed the config file from json to yaml, k8s somehow interfered with it and put it's own values into server.port, which broke the server and it couldn't start.